io/net/bonding.py

```diff
@@ -186,7 +185,7 @@ def bond_remove(self, arg1):
             cmd += 'rmmod bonding;'
             cmd += 'ip addr add %s/%s dev %s;ip link set %s up;sleep 5;'\
                    % (self.peer_first_ipinterface, self.net_mask[0],
-                      self.peer_first_interface, self.peer_first_interface)
+                      self.peer_interfaces[0], self.peer_interfaces[0])
```

Comment:
@smruti77 Why not `self.peer_first_interface` ?

Reply:
this interface is getting from peer_ip and that is assign to to bonding interface, at the time of next test the interface is coming as "bondtest" so couldn't assign ip to that interface and peer connection aslo failing.

Reply:
ah ok.